README.rst

```diff
@@ -692,6 +693,8 @@ to the default list of symfony-level fixers:
         ->finder($finder)
     ;
 
+NOTE: `exclude` will work only for directories, so if you need to exclude file, try `notPath`.
```

Comment:
I would just link to symfony/finder docs.
For now it says about some functionallity, but not all. And recreating Symfony docs here is pointless.